packages/api-clients/open-api/bffApi.yml

```diff
@@ -5383,7 +5383,7 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
-  /eservices/templates/{templateId}/instances:
+  /templates/{templateId}/eservices:
```

Comment:
Could you also double-check other routes involving templates or template instances in this file and in the catalog API and adopt this new standard everywhere?
This one, for example, seems to be a duplicate of this but with a GET: https://github.com/pagopa/interop-be-monorepo/pull/1487/files#r1977574513

Reply:
Also, `/eservices/{eServiceId}/descriptors/{descriptorId}/instances/interface:` could be refactored to become `templates/eservices/{eServiceId}/descriptors/{descriptorId}/interface:`

Reply:
In this PR, I only consider these two endpoints. I think that @Viktor-K are working on others. Maybe we can merge our work in this PR